src/app/users/users.module.ts

```diff
@@ -3,9 +3,10 @@ import { CommonModule } from '@angular/common';
 import { AskQuestionComponent } from './ask-question/ask-question.component';
 import { MaterialImportModule } from '../../material-import/material-import.module';
 import { FormsModule, ReactiveFormsModule } from '@angular/forms';
+import { ProfileComponent } from './profile/profile.component';
 
 @NgModule({
-  declarations: [AskQuestionComponent],
+  declarations: [AskQuestionComponent, ProfileComponent],
```

Comment:
Components can be declared in AT MOST ONE module.  Incidentally, we're going to remove this users module and start fresh.